requirements/tests.txt

```diff
@@ -33,7 +35,7 @@ coverage>=4.3.4     ; python_version < '3.5' and python_version >= '3.4'    # Py
 coverage>=5.3.1     ; python_version < '3.4' and python_version >= '2.7'    # Python 2.7
 coverage>=4.5       ; python_version < '2.7' and python_version >= '2.6'    # Python 2.6
 
-codecov>=2.0.15
+# codecov>=2.0.15
```

Comment:
This module was not installing on python 3.6. Looking at the [repo](https://github.com/codecov/codecov-python), it seems the package is deprecated and the github actions integration should be used alone. But I didn't explore this extensively, so the CI may not work.

Reply:
Yes, this package should be removed. I'm working on some maintence updates, rebasing on that (after it is done) should take care of this.

Reply:
I've merged a branch for version 1.1.0, can you rebase this PR on the latest main?